### DISCUSSION TITLE
Why is Vue a dependency?

### DISCUSSION THREAD

**Discussion by jonasgeiler** [beercss/beercss#294] [Q&A] [answered]
I just installed the `beercss` npm package and noticed that my lockfile had several Vue dependencies added to it. I looked into where they were coming from and noticed that `beercss` has `vue` as a dependency. Can anyone tell me why? I can't even find any reference to `vue` in the `beercss` folder in my `node_modules` (except `package.json`)... Should I create an issue for this?

**Comment by leonardorafael** [comment]
Hi @jonasgeiler , 

1) Can you provide an online example of this?
2) Can you try to import only beercss and confirm that this still happen? 
3) What stack/bundler do you use?

Maybe the bundler or another code, points to vue, because we don't have dependencies (and no issues about this before, very strange).

**Reply by jonasgeiler** [reply]
It's literally right there on the npm dependency page: https://www.npmjs.com/package/beercss?activeTab=dependencies

Also in the package.json: https://github.com/beercss/beercss/blob/15104221584466af436a09a817be80865e1d444c/package.json#L18

**Reply by jonasgeiler** [reply]
I hope I'm not hallucinating 😅

**Reply by leonardorafael** [reply, marked as answer]
Oh i see, this is because the homepage was maded with Vue, but this shouldn't be a part of beercss lib. The lib itself is inside the cdn folder and have a separated build. Anyway I think we can add Vue as a dev dependency here.

**Reply by jonasgeiler** [reply]
That would be great - Thanks! 👍🏻 
Maybe at the same time you could also make `material-dynamic-colors` a `peerDependency`? After all these years I still don't exactly know how peer-dependencies work but this seems like a use-case for them, since you would install `material-dynamic-colors` _alongside_ `beercss` when you need it. This could cause versioning problems when you manually install a different `material-dynamic-colors` version than the version that `beercss` has as a dependency. I don't know the exact details but that's what peer-dependencies are for I think 😅
